src/model.rs

```diff
@@ -1391,7 +1390,7 @@ impl RelationshipType {
 	}
 
 	fn decode(value: Value) -> Result<Self> {
-		value.as_u64().and_then(RelationshipType::from_num).ok_or(Error::Decode("Expected valid RelationshipType", value))
+		value.as_u64().and_then(RelationshipType::from_num).ok_or_else(|| Error::Decode("Expected valid RelationshipType", value))
```

Comment:
See https://github.com/Manishearth/rust-clippy/wiki#or_fun_call for why `ok_or_else` is preferred, although I'm not sure it's needed in this example